_articles/building-community.md

```diff
@@ -273,4 +273,4 @@ Your tiebreaker should be a last resort. Divisive issues are an opportunity for
 
 ## Community is the ❤️ of open source
 
-Healthy, thriving communities fuel the thousands of hours poured into open source every week. Many contributors point to other people as the reason for working - or not working - on open source. By learning how to tap into that power constructively, you'll help someone out there have an unforgettable open source experience.
+Healthy, thriving communities fuel the thousands of hours poured into open source every week. Many contributors point to other people as the reason for working - or not working - on open source. By learning how to tap into that power constructively, you'll help someone out there have an unforgettable open source experience. If you're looking for more ways to build your open source community, check out the chapter ["Building and Nurturing a Welcoming and Supportive Community" in OpenSauced's "Becoming a Maintainer" course](https://intro.opensauced.pizza/#/becoming-a-maintainer/building-community?id=building-and-nurturing-a-welcoming-and-supportive-community).
```

Comment:
I'm not sure we want to link to a whole other guide.  This is being discussed.

Reply:
If I may @jmeridth, I have not seen most materials on becoming an open source maintainer presented in a structured way before, and I think alot of people can benefit from it.

Reply:
Our guide does that. I'll get an answer on this tomorrow and let you know. Thank you for your patience.